File: javascript/even-odd.js
// * normal use method
const a = 10;
if (a % 2 == 0) {
  console.log("number is even");
} else {
  console.log("number is odd");
}

// * name function method
function method() {
  const a = 10;
  if (a % 2 == 0) {
    console.log("number is even");
  } else {
    console.log("number is odd");
  }
}
console.log(method());

// * arrow function method
method = () => {
  const a = 10;
  if (a % 2 == 0) {
    console.log("number is even");
  } else {
    console.log("number is odd");
  }
};
console.log(method());

File: c/even-odd.c
#include <stdio.h>
int main()
{
    int a;
    printf("give the number : ") + scanf("%d", &a);
    if (a % 2 == 0)
    {
        printf("number is even %d\n", a);
    }
    else
    {
        printf("number is odd %d\n", a);
    }
    return 0;
}